File: content/5.polympiads2/2.distance-a-la-racine/.codes/empty.java
import java.util.Scanner;

class Main { 
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int nbArrets = sc.nextInt();

		int[] parents = new int[nbArrets];
		for (int i = 0; i < nbArrets; i ++) {
			parents[i] = sc.nextInt();
		}
		
		int[] distancesAuxParents = new int[nbArrets];
		for (int i = 0; i < nbArrets; i ++) {
			distancesAuxParents[i] = sc.nextInt();
		}
		
		// Affichez nbArrets nombres, les distances au centre ville.
	}
}

File: content/polympiads.2/2.distance-a-la-racine/.codes/empty.cpp
#include <iostream>
#include <vector>

using namespace std;

int main () {
    int nbArrets;
    cin >> nbArrets;

    vector<int> parents(nbArrets);
    for (int i = 0; i < nbArrets; i ++) {
        cin >> parents[i];
    }

    vector<int> distancesAuxParents (nbArrets);
    for (int i = 0; i < nbArrets; i ++) {
        cin >> distancesAuxParents[i];
    }
    
    // Affichez nbNoeuds nombres, les distances au centre ville.
}
